packages/rip/build.sh

```diff
@@ -1,18 +0,0 @@
-TERMUX_PKG_HOMEPAGE=https://github.com/nivekuil/rip
```

Comment:
We should move this one to `disabled-packages`.

Reply:
It's a direct replacement.
I don't think there's any need to keep it in `disabled-packages`.
We didn't do that for `exa` either.

Reply:
I am not so sure about that. Because this package will have different name and will be treated as separate package by CI and our repos.

Reply:
You are right about direct replacement, we do not need to move it to `disabled-packages`.

Reply:
Yeah, same thing as with `exa` and `eza`.
- #17828